app/controllers/pages_controller.rb

```diff
@@ -3,8 +3,9 @@ def home
   end
 
   def dashboard
```

Comment:
def dashboard
    skip_authorization
    # I booked
    @make_bookings = current_user.bookings

    # I am booked by => offers I made, and booked by someone
    @booked_by = current_user.bookings_as_owner
    
    # all offers I made
    current_user.offers
  end

Reply:
We also need some has_many...relationship in models